.github/workflows/memory-leak-testing.yml

```diff
@@ -25,23 +25,20 @@ jobs:
           path: node_modules
           key: ${{ runner.os }}-dependencies-${{ hashFiles('**/pnpm-lock.yaml') }}
           restore-keys: |
-            ${{ runner.os }}-dependencies-
+            ${{ runner.os }}-dependencies-    
 
       - name: Install pnpm
         run: npm install -g pnpm
 
-      - name: Install memlab
-        run: npm install -g memlab
-
       - name: Install dependencies
-        run: pnpm i --frozen-lockfile
+        run: make install
         if: |
           steps.cache-pnpm-dependencies.outputs.cache-hit != 'true' ||
           steps.cache-pnpm-packages.outputs.cache-hit != 'true'
 
       - name: Run dev and memory leak testing
         run: |
-          pnpm run dev &
+          pnpm run dev:en &
```

Comment:
this is hardcode, need to find another option, in scripts we don't need this options

Reply:
fixed

Reply:
you removed from package.json but not from here, fo why you write this comment if this code still there?

Reply:
because I will remove it anyway when I fix the memory leak tests tomorrow.